elastic/kibana
# [Logs onboarding] Getting elastic-agent version from fleet

Closes https://github.com/elastic/kibana/issues/165657.

In https://github.com/elastic/kibana/pull/166811 we exposed the value of latest agent version available, with this PR we are aiming to use that value as the `elastic-agent` version used in the installation script of onboarding flow.

### How to test
1. Enter [System logs onboarding](https://yngrdyn-deploy-kiban-pr166920.kb.us-west2.gcp.elastic-cloud.com/app/observabilityOnboarding/systemLogs)
2. Verify the elastic agent version in the installation script
<img width="870" alt="image" src="https://github.com/elastic/kibana/assets/1313018/c2d99244-6e5a-4d2e-a3fd-eea6041636d2">

3. Go to the [console](https://yngrdyn-deploy-kiban-pr166920.kb.us-west2.gcp.elastic-cloud.com/app/dev_tools#/console)
4. Execute `GET /`
5. Verify Kibana version

As you can see the kibana version is an snapshot but the elastic agent version proposed is the latest one released `8.10.1` allowing us to construct a valid download url like https://artifacts.elastic.co/downloads/beats/elastic-agent/8.10.1-linux-x86_64.tar.gz
